src/Abstractions/Game.cs

```diff
@@ -4,7 +4,7 @@ namespace UltimateTicTacToe.Abstractions
 {
     public class Game
     {
-        public Guid Id { get; set; }
+        public string Id { get; set; }
         public Player? Winner { get; set; }
```

Comment:
So: 
draw and not finished: winner = null.
but with a draw the `FinishedAt` date is set aswell?

Reply:
the idea would be to do it that way. Made a note in #3 to avoid forgetting it.